dll/win32/shimgvw/shimgvw.c

```diff
@@ -1098,6 +1174,8 @@ ImageView_CreateWindow(HWND hwnd, LPCWSTR szFileName)
 
     InitCommonControlsEx(&Icc);
 
+    CoInitializeEx(NULL, COINIT_APARTMENTTHREADED | COINIT_DISABLE_OLE1DDE);
```

Comment:
Hm... I don't see how you use COM here yet. :thinking:

Reply:
https://docs.microsoft.com/en-us/windows/win32/api/shellapi/nf-shellapi-shellexecuteexa

Reply:
![immagine](https://user-images.githubusercontent.com/30959007/108625485-7fc63f00-744b-11eb-8102-9d329a8ba500.png)

Reply:
Ah I see, alright then.

Reply:
‘CoUninitialize` call and return value check is needed.